### PR TITLE
Wishlist management page made

### PR DESCRIPTION
**Pull request by Asypher00**
check wishlist.css for code
Screenshot (23)

### CODE REVIEW

File: wishlist.css [hunk 1]
```diff
@@ -0,0 +1,82 @@
+body{background-color: white;
+    color : black;
+    font-family: "comic sans ms";
+    
+}
+.container{
+    display: flex;
+    flex-direction: column;
+    flex-wrap: wrap;
+    align-items: center;
```
Comment (nakulsharma15) [inline wishlist.css:16]: You can extract this in a utility class so that it can be used anywhere else as well

File: wishlist.css [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+    width : 20%;
+}
+.btn{
+    margin : 0px;
+    background-color: lightgray;
+}
+
+.nav-main{display: flex;
+    justify-content: center;
+    align-items: center;
```
Comment (nakulsharma15) [inline wishlist.css]: Extract this in a utility class for reusability

File: wishlist.css [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+    margin-left : 250%;
+    
+}
+
+.nav-input{text-align: center;
+    font-family: "comic sans ms";
+    color : black;
+    font-size: 1.25rem;
+    width : 500px;
```
Comment (nakulsharma15) [inline wishlist.css]: You can use rem here instead of px

File: wishlist.css [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+   .border{
+    border : 2px solid black;
+    border-radius: 10px;
+   }
+   .wishlist-header{
+       font-size : 3rem;
+       margin : 1.25rem;
+      padding : 10px;
+       background-color: rgb(188, 251, 62);
+       border-radius: 20px;
```
Comment (nakulsharma15) [inline wishlist.css]: You can use rem here instead of px

**Review by nakulsharma15 — APPROVED** [review]
Suggested some minor changes

**Review by rahulcs754 — APPROVED** [review]
please try prettier to align your code.

**Pull request merged**